### PR TITLE
Fix Combination filter in stock management section

### PR DESCRIPTION
**Pull request by fatmaBouchekoua**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.6.1.x
| Description?  | When you do a search by name or by reference, or sort combinations it shows "No records found".
| Type?         | bug fix
| Category?     | FO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | http://forge.prestashop.com/browse/PSCSX-8859
| How to test?  | In the Stock > Stock Management, view details of product that have combinations and use search and sort options.

### CODE REVIEW

File: controllers/admin/AdminStockManagementController.php [hunk 1]
```diff
@@ -176,7 +176,20 @@ public function renderDetails()
             $this->_where = 'AND a.id_product = '.$product_id;
             $this->_group = 'GROUP BY a.id_product_attribute';
 
-            self::$currentIndex = self::$currentIndex.'&id_product='.(int)$product_id;
+            $this->fields_list ['name'] =
```
Comment (maximebiloe) [inline controllers/admin/AdminStockManagementController.php]: Could you remove the space between `fields_list` and `['name']`, please?

File: controllers/admin/AdminStockManagementController.php [hunk 2]
```diff
@@ -870,6 +883,13 @@ public function postProcess()
             $redirect = self::$currentIndex.'&token='.$token;
         }
 
+        if(Tools::isSubmit('submitFilter') && Tools::getIsset('detailsproduct')){
```
Comment (maximebiloe) [inline controllers/admin/AdminStockManagementController.php]: A little space here, please

File: controllers/admin/AdminStockManagementController.php [hunk 3]
```diff
@@ -1349,6 +1369,10 @@ public function displayPrepareRemovestockLink($token = null, $id)
             self::$cache_lang['RemoveStock'] = $this->l('Remove stock');
         }
 
+        if(Tools::getIsset('detailsproduct')) {
```
Comment (maximebiloe) [inline controllers/admin/AdminStockManagementController.php]: A space here too 😉

**Review by maximebiloe — CHANGES_REQUESTED** [review]

**Comment by fatmaBouchekoua** [comment]
Hello @maximebiloe , Done !
Thank you

**Review by maximebiloe — APPROVED** [review]

**Comment by maximebiloe** [comment]
Thank you @fatmaBouchekoua

**Pull request merged**